hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/gas/DispatchType.java

```diff
@@ -30,6 +30,7 @@
 /**
  * Enumerates the types of child transactions that can be dispatched to the HTS system contract.
  */
+@SuppressWarnings("MissingJavadoc")
```

Comment:
Odd, why is this necessary?

Reply:
I added this because I thought the variables are self-explanatory and did not need javadocs but I will add some useful comments